packages/kbn-ui-framework/src/components/button/button_group/_button_group.scss

```diff
@@ -42,3 +42,10 @@
     text-align: center;
   }
 }
+
+// KuiButtonGroup doesn't pass down className, so can't create custom classname
```

Comment:
Not sure if you want to wait, but I can make a PR that removes https://github.com/elastic/kibana/blob/master/src/core_plugins/kibana/public/home/components/tutorial/radio_button_group.js and just uses EuiButtonGroup. That has all of the functionality needed and `RadioButtonGroup` is really just extra baggage now that EUI has this component.